conf/evolutions/bakery_db/1.sql

```diff
@@ -0,0 +1,18 @@
+-- bakery_db schema
+
+-- !Ups
+
+CREATE TABLE IF NOT EXISTS Bakery
```

Comment:
not really a `Bakery`, but a `Product` table for the bakery database

Reply:
Yeh, its not the whole bakery but just the product of it, that makes sense, will change.